libs/blocks/table/table.js

```diff
@@ -533,9 +509,20 @@ export default function init(el) {
       deviceBySize = defineDeviceByScreenSize();
       handleResize();
     });
+
+    isDecorated = true;
   };
 
   window.addEventListener(MILO_EVENTS.DEFERRED, () => {
     handleTable();
   }, true);
+
+  const observer = new window.IntersectionObserver((entries) => {
```

Comment:
Utils has an intersection observer wrapper

Reply:
Hey @mokimo! Thanks for the note. A few more details on this: this particular intersection observer is indeed replaceable with the one from `utils.js`. However, this file has another one [here](https://github.com/adobecom/milo/blob/b048123a0207264b5af083a55f74ed13a0df1737/libs/blocks/table/table.js#L306-L309). That one, unfortunately, can't be replaced with the one from `utils.js`; that is because it depends on being able to tell whether the entry is currently intersecting or not to toggle a class. The callback for `createIntersectionObserver` [here](https://github.com/adobecom/milo/blob/b048123a0207264b5af083a55f74ed13a0df1737/libs/utils/utils.js#L1233-L1235) is only called if the entry is intersecting, so the previously-mentioned toggle will never occur.

With that in mind, what do you choose between:
1. importing the `createIntersectionObserver` method from `utils.js` and using it in one place, while the other place keeps using a regular intersection observer? It could be argued that we're already importing `utils.js`, so there's no overhead in using the method it supplies where we can;
2. we keep things as they are now keep it consistent in the `table.js` file.

Reply:
Let's keep things consistent and the way they are right now